addons/im_livechat/models/res_users.py

```diff
@@ -35,6 +38,17 @@ class ResUsers(models.Model):
         store=False,
         help="When forwarding live chat conversations, the chatbot will prioritize users with matching expertise.",
     )
+    livechat_ongoing_session_count = fields.Integer(
+        "Number of Ongoing sessions",
+        compute="_compute_livechat_ongoing_session_count",
+        groups="im_livechat.im_livechat_group_user,base.group_erp_manager",
```

Comment:
Do we need group_erp_manager here? Isn't admin having access regardless?

Reply:
Indeed, was there for consistency with other fields but not striclty needed. Removed.